### PR TITLE
:bug: remove rest of Fira Code

### LINKED ISSUE #4481: Removing fira-code typeface
Still visible in certain locations. As an example, copy to clipboard, etc. image

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4481
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Can't tell whether it's changing something in the SVG, best visible here: 

Before: 
image  

After: 
image

### CODE REVIEW

File: styles/components/_carousel.scss [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
     &-arrow {
       color: black;
-      font-family: 'Fira Code', monospace;
+      font-family: 'Work Sans', monospace;
```
Comment (prachi00) [inline styles/components/_carousel.scss:57]: I know that this is how we are replacing the font-family right now, but isnt it time to just replace it in html tag once and remove these from all the other css files?

Comment (roiLeo) [reply styles/components/_carousel.scss:57]: I agree with that, we can try to define [Bulma sass variable](https://bulma.io/documentation/customize/variables/#generic-variables) or maybe customize [font family](https://bulma.io/documentation/helpers/typography-helpers/#font-family)

Comment (petersopko) [reply styles/components/_carousel.scss]: I've tried to follow the [Bulma sass vairable](https://bulma.io/documentation/customize/variables/#generic-variables) way, lmk if something needs to be changed.

**Review by roiLeo — COMMENTED** [review]
Screenshot 2022-12-15 at 09-30-17 KodaDot - Kusama NFT Market Explorer

right arrow should use Fira Code if I'm not mistaken

**Comment by petersopko** [comment]
> Screenshot 2022-12-15 at 09-30-17 KodaDot - Kusama NFT Market Explorer
> 
> right arrow should use Fira Code if I'm not mistaken

@exezbcz are we keeping Fira Code on arrow?

**Comment by exezbcz** [comment]
> > Screenshot 2022-12-15 at 09-30-17 KodaDot - Kusama NFT Market Explorer
> > right arrow should use Fira Code if I'm not mistaken
> 
> @exezbcz are we keeping Fira Code on arrow?

i think we can keep it, i just see it bit differently: 
image
issue with work sans is that the typeface doesn't have ligatures
the one in the image i think works fine.

**Comment by petersopko** [comment]
@exezbcz which one of those images do you mean? 

- [ ] @roiLeo's screenshot - Work Sans, can be checked at https://deploy-preview-4488--koda-nuxt.netlify.app/
- [ ] your screenshot - Fira Code, everywhere else.

**Comment by exezbcz** [comment]
> @exezbcz which one of those images do you mean?
> 
> * [ ]  @roiLeo's screenshot - Work Sans, can be checked at https://deploy-preview-4488--koda-nuxt.netlify.app/
> * [ ]  your screenshot - Fira Code, everywhere else.

i prefer the line solid, not divided into --->

**Comment by yangwao** [comment]
pay 50 usd

Resolve merge conflicts and it's good to go

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 26.53 USD/KSM ~ 1.885 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0xb4140b175d01ad646bbd93851f430dc485598f0bae6507c8669cdf0a6d004ad8](https://kusama.subscan.io/extrinsic/0xb4140b175d01ad646bbd93851f430dc485598f0bae6507c8669cdf0a6d004ad8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
cut text at some length on landing page?  
can be done once #4488 merged
image

**Comment by petersopko** [comment]
> [0xb4140b175d01ad646bbd93851f430dc485598f0bae6507c8669cdf0a6d004ad8](https://kusama.subscan.io/extrinsic/0xb4140b175d01ad646bbd93851f430dc485598f0bae6507c8669cdf0a6d004ad8)

conflicts resolved, transaction not 😆

**Comment by yangwao** [comment]
oh probably handling check balance before sending would be nice to preserve close to reality state 

image

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 25.2 USD/KSM ~ 1.984 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0xfcb10abf473e13a34f4da391a6db0c54e4483eabcf3b1f73cff46b4a39c04661](https://kusama.subscan.io/extrinsic/0xfcb10abf473e13a34f4da391a6db0c54e4483eabcf3b1f73cff46b4a39c04661)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**